modules/cnv-deploying-cnv.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_install/installing-container-native-virtualization.adoc
+
+[id="cnv-deploying-cnv_{context}"]
+= Deploying {ProductName}
+
+After subscribing to the *KubeVirt HyperConverged Cluster Operator* catalog,
+create the *KubeVirt HyperConverged Cluster Operator Deployment* custom resource
+to deploy {ProductName}.
+
+.Prerequisites
+
+* You must be subscribed to the *KubeVirt HyperConverged Cluster Operator* catalog
```

Comment:
I'd make all of your prereqs parallel. Sometimes they're "you" statements, and other times they just state the existence of a state. Compare this one to the assembly prereqs.